pkg/k8shandler/certificates.go

```diff
@@ -45,6 +50,72 @@ func extractSecretToFile(namespace string, secretName string, key string, toFile
 	return nil
 }
 
+func writeSecret(logging *v1alpha1.ClusterLogging) (err error) {
+
+	secret := utils.Secret(
+		"logging-master-ca",
```

Comment:
Do we need to still call this 'logging-master-ca'? Is there a way to be more descriptive and drop the 'logging'?  Is it really named correctly if we are including ca, cert, keys?

Reply:
renamed to "master-certs"